android/src/main/java/com/pdftron/reactnative/views/DocumentView.java

```diff
@@ -3606,6 +3620,28 @@ public WritableArray getAllFields(int pageNumber) {
         return null;
     }
 
+    public WritableArray getFieldsForPage(int pageNumber, WritableArray fieldsArray){
```

Comment:
1. formatting
2. in this case since it's updating the array, i don't think it needs to return itself

Reply:
done